database/install-functions.sh

```diff
@@ -29,43 +31,45 @@ function create-functions {
     database=$DATABASE_NAME
   fi
 
+  psql_cmd="psql $database -q -v ON_ERROR_STOP=1"
```

Comment:
I could also define `psql_cmd` as `"psql -q -v ON_ERROR_STOP"` at the top of each file because that's what is used for everything, regardless of database. The way I did it here, it ends up varying (`install.sh` is different because it installs things to `postgres`).